Tests/test_imagefont.py

```diff
@@ -164,7 +175,7 @@ def test_render_multiline_text(self):
             # to multiline_text()
             draw.text((0, 0), TEST_TEXT, fill=None, font=ttf, anchor=None,
                       spacing=4, align="left")
-            draw.text((0, 0), TEST_TEXT, None, ttf, None, 4, "left")
+            draw.text((0, 0), TEST_TEXT, None, ttf, None, None, 4, "left")
```

Comment:
Got it.

I've gone ahead and refactored the ImageDraw.text() method signature to no longer take in **kwargs which I believe will allow for greater flexibility in future.

Other than passing the Align and Spacing args to multiline_text, I don't believe it's using anything else (Atleast it's not documented)
Moved to explicitly declaring the keyword arguments